src/context/quiz-context.jsx

```diff
@@ -0,0 +1,189 @@
+import { createContext, useContext, useReducer } from "react";
+
+const QuizContext = createContext();
+
+const initialState = {
+  quizData1: [
```

Comment:
just a suggestion:  you can make a separate file for question's data

Reply:
Will see this